modules/cloudtrail/main.tf

```diff
@@ -0,0 +1,328 @@
+data "aws_caller_identity" "current" {}
```

Comment:
Every name within this module should be called `cloudtrail` instead of `ksoc_discovery`